### PR TITLE
WP: Architecture part with the last ideas about logic extension.

### PR DESCRIPTION
**Pull request by IlyaMoskva**
Issue #21 solved.

### CODE REVIEW

File: white-paper/wp.tex [hunk 1]
```diff
@@ -222,31 +222,38 @@ \section{Architecture}
...
     \end{itemize}
-    Edges of graph are routes from one Node to another. They must have their own configuration properties:
+    \textbf{Edges} of graph represent network connections from one Node to another. They must have their own configuration properties:
     \begin{itemize}
-        \item Throughput (max Mb/sec)
+        \item Throughput (max Mb/sec)\todo Or it can be on vertex level only.
         \item State (Active/Down)
     \end{itemize}
+    If edge is not active, system must find another way to get the target node.\par
+    In corner case there may be nodes without GitHub repositories, serve as \textbf{routers}. Such routers can have different strategies and have not storage (can't serve as a final destination of the request.\par
```
Comment (g4s8) [inline white-paper/wp.tex:237]: @IlyaMoskva I don't think we need to mention repositories or Github here - a server may run any arbitrary logic code, not related to git ropos

File: white-paper/wp.tex [hunk 2]
```diff
@@ -222,31 +222,38 @@ \section{Architecture}
...
     Node state must be set with given distribution method. (Normal, random, etc)\par
-    Repository is a logic component, represents GitHub repository instance. Should be distributed by at least three Nodes, connected with one sub-graph. Nodes must be connected either directly or not.
-    
+    Repository is a logic component, represents GitHub repository instance. Should be distributed by at least three Nodes, connected with one sub-graph. Nodes must be connected either directly or not.\par
```
Comment (g4s8) [inline white-paper/wp.tex:239]: @IlyaMoskva and same here -- this tool theoretically could be used to simulate any other distributed network, not related to git repositories

**Review by g4s8 — CHANGES_REQUESTED** [review]
@IlyaMoskva please check my comments

File: white-paper/wp.tex (hunk 2 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex:239]: @g4s8 , I agree, changed.

File: white-paper/wp.tex (hunk 1 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex:237]: Yes, @g4s8 , good point, I've changed the text.

**Comment by IlyaMoskva** [comment]
Uploaded forgotten file with updated bib list.

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva thanks

**Pull request merged**